app/src/main/java/org/oppia/android/app/splash/SplashActivityPresenter.kt

```diff
@@ -83,6 +100,25 @@ class SplashActivityPresenter @Inject constructor(
     )
   }
 
+  private fun getParameterLoadingStatus(): LiveData<Boolean> {
```

Comment:
`getParameterLoadingStatus` sound like a UI process. I'd rename this to something like `fetchPlatformParametersFromDatabase` and then `processParameterLoadingStatus` => `processPlatformParameters` or something to it's clearer that it's data processing.

Reply:
Done